backend/src/api/Data/DTO/UserDTO.cs

```diff
@@ -2,12 +2,11 @@
 
 namespace api.Data.DTO;
 
-public class UserDTO
+public class UserDto
```

Comment:
Please keep the class and file name in sync (I do not mind if it is Dto or DTO, but please be consistant)

Reply:
Fixed, I reverted back to DTO.